public/app/features/dashboard-scene/settings/annotations/AnnotationSettingsEdit.tsx

```diff
@@ -0,0 +1,339 @@
+        >
+          <Checkbox
+            name="hide"
+            id="hide"
+            value={annotation.hide}
+            onChange={onChange}
+            data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.hide}
+          />
+        </Field>
+        <Field label="Color" description="Color to use for the annotation event markers">
+          <HorizontalGroup>
+            <ColorValueEditor value={annotation?.iconColor} onChange={onColorChange} />
+          </HorizontalGroup>
+        </Field>
+        <Field label="Show in" data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.showInLabel}>
+          <>
+            <Select
+              options={panelFilters}
+              value={panelFilter}
+              onChange={onFilterTypeChange}
+              data-testid={selectors.components.Annotations.annotationsTypeInput}
+            />
+            {panelFilter !== PanelFilterType.AllPanels && (
+              <MultiSelect
+                options={selectablePanels}
+                value={selectablePanels.filter((panel) => annotation.filter?.ids.includes(panel.value!))}
+                onChange={onAddFilterPanelID}
+                isClearable={true}
+                placeholder="Choose panels"
+                width={100}
+                closeMenuOnSelect={false}
+                className={styles.select}
+                data-testid={selectors.components.Annotations.annotationsChoosePanelInput}
+              />
+            )}
+          </>
+        </Field>
+      </FieldSet>
+      <FieldSet>
+        <h3 className="page-heading">Query</h3>
+        {ds?.annotations && dsi && (
+          <StandardAnnotationQueryEditor
+            datasource={ds}
+            datasourceInstanceSettings={dsi}
+            annotation={annotation}
+            onChange={(annotation) => onUpdate(annotation, editIndex)}
+          />
+        )}
+        {ds && !ds.annotations && (
+          <AngularEditorLoader
+            datasource={ds}
+            annotation={annotation}
+            onChange={(annotation) => onUpdate(annotation, editIndex)}
+          />
+        )}
+      </FieldSet>
+      <Stack>
+        {!annotation.builtIn && (
+          <Button
+            variant="destructive"
+            onClick={onDeleteAndLeavePage}
+            data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.delete}
+          >
+            Delete
+          </Button>
+        )}
+        <Button
+          variant="secondary"
+          onClick={onPreview}
+          data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.previewInDashboard}
+        >
+          Preview in dashboard
+        </Button>
+        <Button
+          variant="primary"
+          onClick={onApply}
+          data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.apply}
+        >
+          Apply
+        </Button>
```

Comment:
We can replace both buttons with "Back to list", since the new nav bar contains `Back to dashboard`, the same as the preview dashboard. Also, we agreed with @Ijin08 to not have apply buttons and have back-to-list ones.

As we do for variables (you will see links keeps the apply, I'm opening a PR to fix it)
<img width="1712" alt="Captura de pantalla 2024-02-07 a las 8 56 00" src="https://github.com/grafana/grafana/assets/5699976/39738312-3328-4f13-b392-9f9f97942dbb">

```suggestion
        <Button
          onClick={onBackToList}
          data-testid={selectors.pages.Dashboard.Settings.Annotations.NewAnnotation.apply}
        >
          Back to list
        </Button>
```